docs/source/openfl/components.rst

```diff
@@ -51,41 +56,48 @@ The Collaborator is a short-lived entity that manages training the model on loca
     - converting deep learning framework-specific tensor objects to |productName| inner representation, and
     - exchanging model parameters with the Aggregator.
 
-The Collaborator is created by the :ref:`Envoy <openfl_ll_components_envoy>` when a new experiment is submitted. 
+The Collaborator is created by the :ref:`Envoy <openfl_ll_components_envoy>` when a new experiment is submitted 
+in the :ref:`Director-based workflow <director_workflow>`. The Collaborator should be started from CLI if a user follows the 
+:ref:`Aggregator-based workflow <running_the_federation_aggregator_based>`
 
-Every Collaborator is a unique service. The model is loaded with a local *shard descriptor* to perform tasks included in an FL experiment. At the end of the training task, weights are extracted and sent to the central node and aggregated.
+Every Collaborator is a unique service. The data loader is loaded with a local *shard descriptor* to perform tasks 
+included in an FL experiment. At the end of the training task, weights are extracted and sent to the central node 
+and aggregated.
 
-Converting tensor objects is handled by :ref:`framework adapter <framework_adapter>` plugins. Included in |productName| are framework adapters for PyTorch and TensorFlow 2.x. These framework adapters are extensible. You can contribute new framework adapters for deep learning frameworks you would like see supported in |productName|. 
+Converting tensor objects is handled by :ref:`framework adapter <framework_adapter>` plugins. 
+Included in |productName| are framework adapters for PyTorch and TensorFlow 2.x. 
+The list of framework adapters is extensible. User can contribute new framework adapters for deep learning frameworks 
```

Comment:
`User can contribute ... they would like see ...` or  `You can contribute ... you would like see ...`

Reply:
fixed thanks